pkgs/development/python-modules/johnnycanencrypt/default.nix

```
@@ -39,7 +39,7 @@ buildPythonPackage rec {
 
   cargoSha256 = "0ifvpdizcdp2c5x2x2j1bhhy5a75q0pk7a63dmh52mlpmh45fy6r";
 
-  LIBCLANG_PATH = llvmPackages.libclang + "/lib";
+  LIBCLANG_PATH = llvmPackages.libclang.lib + "/lib";
```

```suggestion
  LIBCLANG_PATH = "${llvmPackages.libclang}/lib";
```

Shouldn't this auto detect the right outout?

---

only when declared in something like `buildInputs`. This is just constructing a path.